packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/ConfirmBulkActionDialog/tests/index.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import ConfirmBulkActionDialog from '..';
+
+describe('ConfirmBulkActionDialog', () => {
+  const Component = (props) => (
+    <ConfirmBulkActionDialog
+      isOpen={false}
+      onToggleDialog={jest.fn()}
+      dialogBody={<div data-testid="dialog-body" />}
+      endAction={<div data-testid="end-action" />}
```

Comment:
copilot gave me this suggestion 🤔 . Since the test ids are defined here in the test I thought why not. wdyt? I know it's last on the list in terms of priority https://testing-library.com/docs/queries/about/#priority

Reply:
Fine by me since it doesn't affect the actual component 👍